pkg/mkimage-raw-efi/install

```diff
@@ -363,24 +400,31 @@ YEAR=$(date +%Y)
 if [ "$YEAR" -gt 2020 ] && [ ! -f /config/device.cert.pem ]; then
    if [ -c $TPM_DEVICE_PATH ] && ! [ -f /config/disable-tpm ]; then
       echo "Generating TPM device certificate"
+      logmsg "Generating TPM device certificate"
```

Comment:
Seems we will have two lines in console here, should we remove one of them?
```
No TPM; Generating soft device certificate
2022-02-23 15:06:21 : No TPM; Generating soft device certificate
```

Reply:
+1
I think there are several of those where there is an echo and logmsg with the same string and we only need the logmsg one.

Reply:
Yep agree 100%. I will replace the echo with logmsg and update the PR.